pom.xml

```diff
@@ -63,6 +65,7 @@
         <module>task</module>
         <module>client</module>
         <module>examples</module>
+        <module>commons</module>
```

Comment:
As runtime is dependent on commons dependencies (jackson), asked to move it up. You can ignore this, as jackson scope is test in commons.